script/import_db.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
No, sorry, I didn't mean to suggest you modify or move create_test_db.sh at all. I meant keep the existing import_db script you had, and call it when the container starts if an environment variable is set. Currently we have docker/musicbrainz-test-database/create_test_db.sh run *when the image is built*, but you of course shouldn't import dumps when building the image, only after the container starts. There will already be an (empty) database at that point, but you can just use `dropdb` first.